utils/python/CIME/SystemTests/erio.py

```diff
@@ -109,7 +68,9 @@ def run_phase(self):
 
         for idx, pio_type1 in enumerate(self._pio_types):
             if pio_type1 != "default":
+                self._case.set_value("PIO_TYPENAME", pio_type1)
```

Comment:
Here is where you set the various pio_types.   One concern I have - the netcdf4c format is a lossy compressed format, I would expect that output not to compare to the others with cprnc - It seems to pass for X and A cases though - I am trying some tests with external components.